brands/shop/mobile_phone.json

```diff
@@ -40,6 +40,16 @@
       "shop": "mobile_phone"
     }
   },
+  "shop/mobile_phone|Best Buy Mobile": {
+    "countryCodes": ["ca"],
```

Comment:
These are in the US as well

Reply:
According to https://www.theverge.com/2018/3/1/17066232/best-buy-close-all-mobile-stores-may-us, the American ones closed in 2018.  Are there still a few stragglers left?

Reply:
There's a ton of leftovers. I didn't realize they had closed them all down. I should clean those up sometime. Thanks for the heads up.